apps/mobile/src/queries/transaction/use-bitcoin-broadcast-transaction.ts

```diff
@@ -3,10 +3,11 @@ import { useCallback, useState } from 'react';
 import { TransactionInput } from '@scure/btc-signer/psbt';
 
 import { decodeBitcoinTx } from '@leather.io/bitcoin';
+import { filterOutIntentionalUtxoSpend } from '@leather.io/query';
 import { delay } from '@leather.io/utils';
 
 import { useBitcoinClient } from '../clients/bitcoin-client';
-import { filterOutIntentionalUtxoSpend, useCheckUnspendableUtxos } from './use-check-utxos';
+import { useCheckUnspendableUtxos } from './use-check-utxos';
```

Comment:
You should add a log for the silent return on L45 here. I have a PR up, but will close it bc of this work.
https://github.com/leather-io/mono/pull/988